hr_period/README.rst

```diff
@@ -0,0 +1,91 @@
+HR Payroll Period
```

Comment:
Please refer to https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst for last version of the template